io.github.cudatext.CudaText-Qt.yml

```diff
@@ -0,0 +1,188 @@
+app-id: io.github.cudatext.CudaText-Qt
+runtime: org.kde.Platform
+runtime-version: '6.6'
+sdk: org.kde.Sdk
+build-options:
+  append-path: /app/lib/sdk/freepascal/bin
+  env:
+    PPC_CONFIG_PATH: /app/lib/sdk/freepascal/etc
+    FPCDIR: /app/lib/sdk/freepascal/fpcsrc
+    FPC_DIR: /app/lib/sdk/freepascal/fpcsrc
+    LAZARUSDIR: /app/lib/sdk/freepascal/share/lazarus
+cleanup-commands:
+  - rm -rf /app/lib/sdk/freepascal
+default-branch: stable
+command: cudatext
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --filesystem=home
```

Comment:
why do you need home?

Reply:
Being a general text editor, the user might use it to open any text (or html, or code source etc) files that belong to them, so it doesn't make much sense to restrict it further than this IMO, although I suppose `xdg-documents` might be enough for most users.